contributions/catalog/controllers/contribute.py

```diff
@@ -21,6 +22,7 @@
     Blueprint, render_template, request, session, redirect, url_for
 )
 from requests_oauthlib import OAuth2Session
+from formencode import variabledecode
```

Comment:
This needs to be added to requirements.txt. Thanks.

Reply:
added